{"asctime": "2020-03-27 16:34:03,069", "levelname": "DEBUG", "message": "ApplyDataModel :Class:ApplyDataModel"}
{"asctime": "2020-03-27 16:34:03,073", "levelname": "DEBUG", "message": "ApplyDataModel :loaded config for ApplyDataModel from c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.json"}
{"asctime": "2020-03-27 16:34:03,074", "levelname": "DEBUG", "message": "ApplyDataModel :c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel/schemas"}
{"asctime": "2020-03-27 16:34:03,075", "levelname": "DEBUG", "message": "ApplyDataModel :c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel/schemas/prjs/prjs.gdb"}
{"asctime": "2020-03-27 16:34:33,498", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain AssociatedStructType"}
{"asctime": "2020-03-27 16:34:33,499", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:33,499", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:35,084", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain TrueFalse"}
{"asctime": "2020-03-27 16:34:35,084", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:35,084", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:36,723", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain FeederCableSize"}
{"asctime": "2020-03-27 16:34:36,723", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:36,723", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:40,095", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain DistributionCableSize"}
{"asctime": "2020-03-27 16:34:40,095", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:40,095", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:43,372", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain DropCableSize"}
{"asctime": "2020-03-27 16:34:43,373", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:43,373", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:44,196", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain TerminalCableSize"}
{"asctime": "2020-03-27 16:34:44,196", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:44,197", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:46,957", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain TerminalCableType"}
{"asctime": "2020-03-27 16:34:46,958", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:46,958", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:47,749", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain FeederCableType"}
{"asctime": "2020-03-27 16:34:47,749", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:47,749", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:48,604", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain DistributionCableType"}
{"asctime": "2020-03-27 16:34:48,604", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:48,604", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:49,461", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain DropCableType"}
{"asctime": "2020-03-27 16:34:49,462", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:49,462", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:50,324", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain DipType"}
{"asctime": "2020-03-27 16:34:50,325", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:50,325", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:51,205", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain StoredSlackLength"}
{"asctime": "2020-03-27 16:34:51,205", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:51,206", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:54,756", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain SplitterType"}
{"asctime": "2020-03-27 16:34:54,756", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:54,757", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:59,500", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain FiberTerminalType"}
{"asctime": "2020-03-27 16:34:59,501", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:34:59,502", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:04,084", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain FiberTerminalTail"}
{"asctime": "2020-03-27 16:35:04,084", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:04,085", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:07,891", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain Colour"}
{"asctime": "2020-03-27 16:35:07,892", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:07,892", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:11,768", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain SplitterAreaType"}
{"asctime": "2020-03-27 16:35:11,769", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:11,769", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:13,159", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain TerminalAreaType"}
{"asctime": "2020-03-27 16:35:13,159", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:13,160", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:14,049", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain ConduitInstallationType"}
{"asctime": "2020-03-27 16:35:14,049", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:14,049", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:16,126", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain StrandInstallationType"}
{"asctime": "2020-03-27 16:35:16,126", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:16,126", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:16,934", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain ConduitSize"}
{"asctime": "2020-03-27 16:35:16,934", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:16,935", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:19,040", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain StrandSize"}
{"asctime": "2020-03-27 16:35:19,041", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:19,041", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:19,886", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain PedestalType"}
{"asctime": "2020-03-27 16:35:19,887", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:19,887", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:21,345", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain HandholeType"}
{"asctime": "2020-03-27 16:35:21,345", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:21,346", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:24,127", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain SpliceEnclosureType"}
{"asctime": "2020-03-27 16:35:24,127", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:24,128", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:25,559", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain CabinetType"}
{"asctime": "2020-03-27 16:35:25,560", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:25,560", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:31,317", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain ChamberType"}
{"asctime": "2020-03-27 16:35:31,317", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:31,317", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:32,093", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain PoleType"}
{"asctime": "2020-03-27 16:35:32,093", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:32,094", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:32,901", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain WorkAreaArchitecture"}
{"asctime": "2020-03-27 16:35:32,902", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:32,902", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:34,165", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain WorkAreaCableType"}
{"asctime": "2020-03-27 16:35:34,166", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:34,166", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:35,562", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain WorkAreaStatus"}
{"asctime": "2020-03-27 16:35:35,563", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:35,563", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:39,255", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain StructureAudit"}
{"asctime": "2020-03-27 16:35:39,255", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:39,255", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:35:42,255", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Assembly', 'geometry': '', 'spatialref': '', 'fields': 'assembly.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:35:42,412", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class Assembly exists."}
{"asctime": "2020-03-27 16:35:42,413", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:35:43,706", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table Assembly"}
{"asctime": "2020-03-27 16:35:45,760", "levelname": "DEBUG", "message": "ApplyDataModel :Assembly created"}
{"asctime": "2020-03-27 16:35:45,762", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Assembly"}
{"asctime": "2020-03-27 16:35:45,762", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AssemblyId"}
{"asctime": "2020-03-27 16:35:47,168", "levelname": "DEBUG", "message": "ApplyDataModel :Field AssemblyId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Assembly"}
{"asctime": "2020-03-27 16:35:47,168", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureId"}
{"asctime": "2020-03-27 16:35:48,511", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Assembly"}
{"asctime": "2020-03-27 16:35:48,511", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RUSCode"}
{"asctime": "2020-03-27 16:35:49,892", "levelname": "DEBUG", "message": "ApplyDataModel :Field RUSCode added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Assembly"}
{"asctime": "2020-03-27 16:35:49,893", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'AssociatedStructure', 'geometry': '', 'spatialref': '', 'fields': 'associatedstructure.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:35:50,095", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class AssociatedStructure exists."}
{"asctime": "2020-03-27 16:35:50,095", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:35:51,402", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table AssociatedStructure"}
{"asctime": "2020-03-27 16:35:53,303", "levelname": "DEBUG", "message": "ApplyDataModel :AssociatedStructure created"}
{"asctime": "2020-03-27 16:35:53,304", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:35:53,304", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureId"}
{"asctime": "2020-03-27 16:35:54,631", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:35:54,631", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CableId"}
{"asctime": "2020-03-27 16:35:55,986", "levelname": "DEBUG", "message": "ApplyDataModel :Field CableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:35:55,986", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureType"}
{"asctime": "2020-03-27 16:35:57,542", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:35:57,542", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StartStructure"}
{"asctime": "2020-03-27 16:35:59,169", "levelname": "DEBUG", "message": "ApplyDataModel :Field StartStructure added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:35:59,170", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EndStructure"}
{"asctime": "2020-03-27 16:36:00,895", "levelname": "DEBUG", "message": "ApplyDataModel :Field EndStructure added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/AssociatedStructure"}
{"asctime": "2020-03-27 16:36:00,895", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Cable', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'cable.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:36:01,071", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class Cable exists."}
{"asctime": "2020-03-27 16:36:01,071", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:36:02,267", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Cable"}
{"asctime": "2020-03-27 16:36:04,452", "levelname": "DEBUG", "message": "ApplyDataModel :Cable created"}
{"asctime": "2020-03-27 16:36:04,453", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:04,453", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CableId"}
{"asctime": "2020-03-27 16:36:06,107", "levelname": "DEBUG", "message": "ApplyDataModel :Field CableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:06,107", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-27 16:36:07,782", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:07,782", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SupportingStructureId"}
{"asctime": "2020-03-27 16:36:09,350", "levelname": "DEBUG", "message": "ApplyDataModel :Field SupportingStructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:09,351", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Size"}
{"asctime": "2020-03-27 16:36:10,990", "levelname": "DEBUG", "message": "ApplyDataModel :Field Size added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:10,991", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CableType"}
{"asctime": "2020-03-27 16:36:12,751", "levelname": "DEBUG", "message": "ApplyDataModel :Field CableType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:12,752", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SheathId"}
{"asctime": "2020-03-27 16:36:14,507", "levelname": "DEBUG", "message": "ApplyDataModel :Field SheathId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:14,507", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SectionId"}
{"asctime": "2020-03-27 16:36:16,207", "levelname": "DEBUG", "message": "ApplyDataModel :Field SectionId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:16,207", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-27 16:36:17,989", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:17,990", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:36:19,770", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:19,770", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerServicePointId"}
{"asctime": "2020-03-27 16:36:21,538", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:21,538", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeDetailId"}
{"asctime": "2020-03-27 16:36:23,195", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeDetailId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:23,195", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RangeLabel"}
{"asctime": "2020-03-27 16:36:24,935", "levelname": "DEBUG", "message": "ApplyDataModel :Field RangeLabel added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Cable"}
{"asctime": "2020-03-27 16:36:24,936", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CableFeature', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'cablefeature.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:36:25,173", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class CableFeature exists."}
{"asctime": "2020-03-27 16:36:25,174", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:36:26,414", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CableFeature"}
{"asctime": "2020-03-27 16:36:28,387", "levelname": "DEBUG", "message": "ApplyDataModel :CableFeature created"}
{"asctime": "2020-03-27 16:36:28,388", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:28,388", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:36:30,042", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:30,043", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-27 16:36:31,744", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:31,744", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:36:33,491", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:33,491", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FeatureType"}
{"asctime": "2020-03-27 16:36:35,241", "levelname": "DEBUG", "message": "ApplyDataModel :Field FeatureType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:35,241", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromCableId"}
{"asctime": "2020-03-27 16:36:37,027", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromCableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:37,027", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToCableId"}
{"asctime": "2020-03-27 16:36:38,860", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToCableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CableFeature"}
{"asctime": "2020-03-27 16:36:38,861", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOffice', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'centraloffice.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:36:39,065", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class CentralOffice exists."}
{"asctime": "2020-03-27 16:36:39,065", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:36:40,227", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CentralOffice"}
{"asctime": "2020-03-27 16:36:42,177", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOffice created"}
{"asctime": "2020-03-27 16:36:42,178", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOffice"}
{"asctime": "2020-03-27 16:36:42,178", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-27 16:36:43,761", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOffice"}
{"asctime": "2020-03-27 16:36:43,761", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:36:45,379", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOffice"}
{"asctime": "2020-03-27 16:36:45,379", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOfficeDetail', 'geometry': '', 'spatialref': '', 'fields': 'centralofficedetail.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:36:45,537", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class CentralOfficeDetail exists."}
{"asctime": "2020-03-27 16:36:45,538", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:36:46,678", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:48,572", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOfficeDetail created"}
{"asctime": "2020-03-27 16:36:48,574", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:48,574", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-27 16:36:49,904", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:49,905", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Code"}
{"asctime": "2020-03-27 16:36:51,198", "levelname": "DEBUG", "message": "ApplyDataModel :Field Code added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:51,199", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CircuitNumber"}
{"asctime": "2020-03-27 16:36:52,551", "levelname": "DEBUG", "message": "ApplyDataModel :Field CircuitNumber added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:52,551", "levelname": "DEBUG", "message": "ApplyDataModel :Applying DetailId"}
{"asctime": "2020-03-27 16:36:53,786", "levelname": "DEBUG", "message": "ApplyDataModel :Field DetailId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:53,786", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-27 16:36:55,060", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-27 16:36:55,060", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOfficeArea', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'centralofficearea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:36:55,285", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class CentralOfficeArea exists."}
{"asctime": "2020-03-27 16:36:55,286", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:36:56,446", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CentralOfficeArea"}
{"asctime": "2020-03-27 16:36:58,442", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOfficeArea created"}
{"asctime": "2020-03-27 16:36:58,443", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeArea"}
{"asctime": "2020-03-27 16:36:58,443", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeAreaId"}
{"asctime": "2020-03-27 16:37:00,181", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeAreaId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeArea"}
{"asctime": "2020-03-27 16:37:00,181", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-27 16:37:01,843", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/CentralOfficeArea"}
{"asctime": "2020-03-27 16:37:01,844", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Equipment', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'equipment.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:37:02,022", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class Equipment exists."}
{"asctime": "2020-03-27 16:37:02,022", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:37:03,328", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Equipment"}
{"asctime": "2020-03-27 16:37:05,406", "levelname": "DEBUG", "message": "ApplyDataModel :Equipment created"}
{"asctime": "2020-03-27 16:37:05,407", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:05,408", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:37:07,151", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:07,152", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentId"}
{"asctime": "2020-03-27 16:37:08,931", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:08,932", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentType"}
{"asctime": "2020-03-27 16:37:10,832", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:10,833", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:37:12,567", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:12,567", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-27 16:37:14,229", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:14,229", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RangeLabel"}
{"asctime": "2020-03-27 16:37:16,048", "levelname": "DEBUG", "message": "ApplyDataModel :Field RangeLabel added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:16,049", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Length"}
{"asctime": "2020-03-27 16:37:17,870", "levelname": "DEBUG", "message": "ApplyDataModel :Field Length added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:17,871", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ActualLength"}
{"asctime": "2020-03-27 16:37:19,513", "levelname": "DEBUG", "message": "ApplyDataModel :Field ActualLength added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Equipment"}
{"asctime": "2020-03-27 16:37:19,514", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupLine', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'markupline.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:37:19,735", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class MarkupLine exists."}
{"asctime": "2020-03-27 16:37:19,735", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:37:21,064", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupLine"}
{"asctime": "2020-03-27 16:37:23,171", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupLine created"}
{"asctime": "2020-03-27 16:37:23,172", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupLine"}
{"asctime": "2020-03-27 16:37:23,172", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:37:24,953", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupLine"}
{"asctime": "2020-03-27 16:37:24,954", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:37:26,738", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupLine"}
{"asctime": "2020-03-27 16:37:26,738", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:37:28,691", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupLine"}
{"asctime": "2020-03-27 16:37:28,692", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupPoint', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'markuppoint.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:37:28,936", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class MarkupPoint exists."}
{"asctime": "2020-03-27 16:37:28,937", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:37:30,384", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupPoint"}
{"asctime": "2020-03-27 16:37:32,438", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupPoint created"}
{"asctime": "2020-03-27 16:37:32,438", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPoint"}
{"asctime": "2020-03-27 16:37:32,439", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:37:34,040", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPoint"}
{"asctime": "2020-03-27 16:37:34,040", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:37:35,788", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPoint"}
{"asctime": "2020-03-27 16:37:35,788", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:37:37,888", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPoint"}
{"asctime": "2020-03-27 16:37:37,888", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupPolygon', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'markuppolygon.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:37:38,092", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class MarkupPolygon exists."}
{"asctime": "2020-03-27 16:37:38,092", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:37:39,404", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupPolygon"}
{"asctime": "2020-03-27 16:37:41,407", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupPolygon created"}
{"asctime": "2020-03-27 16:37:41,408", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPolygon"}
{"asctime": "2020-03-27 16:37:41,408", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:37:43,084", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPolygon"}
{"asctime": "2020-03-27 16:37:43,084", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:37:44,882", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPolygon"}
{"asctime": "2020-03-27 16:37:44,882", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:37:46,917", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/MarkupPolygon"}
{"asctime": "2020-03-27 16:37:46,918", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'NetworkConnectivity', 'geometry': '', 'spatialref': '', 'fields': 'networkconnectivity.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:37:47,122", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class NetworkConnectivity exists."}
{"asctime": "2020-03-27 16:37:47,123", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:37:48,274", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:50,352", "levelname": "DEBUG", "message": "ApplyDataModel :NetworkConnectivity created"}
{"asctime": "2020-03-27 16:37:50,353", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:50,354", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ConnectionId"}
{"asctime": "2020-03-27 16:37:51,823", "levelname": "DEBUG", "message": "ApplyDataModel :Field ConnectionId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:51,824", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InFeatureId"}
{"asctime": "2020-03-27 16:37:53,257", "levelname": "DEBUG", "message": "ApplyDataModel :Field InFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:53,258", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InConnectionStart"}
{"asctime": "2020-03-27 16:37:54,669", "levelname": "DEBUG", "message": "ApplyDataModel :Field InConnectionStart added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:54,669", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InConnectionStop"}
{"asctime": "2020-03-27 16:37:56,159", "levelname": "DEBUG", "message": "ApplyDataModel :Field InConnectionStop added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:56,160", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutFeatureId"}
{"asctime": "2020-03-27 16:37:57,664", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:57,665", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutConnectionStart"}
{"asctime": "2020-03-27 16:37:59,008", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutConnectionStart added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:37:59,009", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutConnectionStop"}
{"asctime": "2020-03-27 16:38:00,478", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutConnectionStop added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:38:00,478", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Loss"}
{"asctime": "2020-03-27 16:38:01,908", "levelname": "DEBUG", "message": "ApplyDataModel :Field Loss added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/NetworkConnectivity"}
{"asctime": "2020-03-27 16:38:01,909", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'PrintTile', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'printtile.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:38:02,094", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class PrintTile exists."}
{"asctime": "2020-03-27 16:38:02,094", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:38:03,378", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class PrintTile"}
{"asctime": "2020-03-27 16:38:05,465", "levelname": "DEBUG", "message": "ApplyDataModel :PrintTile created"}
{"asctime": "2020-03-27 16:38:05,466", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PrintTile"}
{"asctime": "2020-03-27 16:38:05,466", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-27 16:38:07,124", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PrintTile"}
{"asctime": "2020-03-27 16:38:07,124", "levelname": "DEBUG", "message": "ApplyDataModel :Applying PageNumber"}
{"asctime": "2020-03-27 16:38:08,798", "levelname": "DEBUG", "message": "ApplyDataModel :Field PageNumber added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PrintTile"}
{"asctime": "2020-03-27 16:38:08,798", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Route', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'route.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:38:09,079", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class Route exists."}
{"asctime": "2020-03-27 16:38:09,080", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:38:10,482", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Route"}
{"asctime": "2020-03-27 16:38:12,615", "levelname": "DEBUG", "message": "ApplyDataModel :Route created"}
{"asctime": "2020-03-27 16:38:12,616", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:12,616", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:38:14,236", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:14,236", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-27 16:38:15,888", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:15,888", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromStructureId"}
{"asctime": "2020-03-27 16:38:17,633", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromStructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:17,633", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToStructureId"}
{"asctime": "2020-03-27 16:38:19,454", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToStructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:19,455", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-27 16:38:21,194", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:21,195", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:38:23,119", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Route"}
{"asctime": "2020-03-27 16:38:23,120", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'ServicePoint', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'servicepoint.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:38:23,323", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class ServicePoint exists."}
{"asctime": "2020-03-27 16:38:23,323", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:38:24,626", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class ServicePoint"}
{"asctime": "2020-03-27 16:38:26,648", "levelname": "DEBUG", "message": "ApplyDataModel :ServicePoint created"}
{"asctime": "2020-03-27 16:38:26,649", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:26,650", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ServicePointId"}
{"asctime": "2020-03-27 16:38:28,426", "levelname": "DEBUG", "message": "ApplyDataModel :Field ServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:28,427", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerNumber"}
{"asctime": "2020-03-27 16:38:30,223", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerNumber added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:30,223", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerServicePointId"}
{"asctime": "2020-03-27 16:38:31,942", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:31,943", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-27 16:38:33,738", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:33,738", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Address"}
{"asctime": "2020-03-27 16:38:35,414", "levelname": "DEBUG", "message": "ApplyDataModel :Field Address added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServicePoint"}
{"asctime": "2020-03-27 16:38:35,415", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'ServingArea', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'servingarea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:38:35,628", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class ServingArea exists."}
{"asctime": "2020-03-27 16:38:35,629", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:38:37,007", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class ServingArea"}
{"asctime": "2020-03-27 16:38:39,144", "levelname": "DEBUG", "message": "ApplyDataModel :ServingArea created"}
{"asctime": "2020-03-27 16:38:39,145", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:39,145", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:38:40,813", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:40,814", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaType"}
{"asctime": "2020-03-27 16:38:42,425", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:42,425", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentId"}
{"asctime": "2020-03-27 16:38:44,259", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:44,259", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-27 16:38:46,023", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:46,023", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeDetailId"}
{"asctime": "2020-03-27 16:38:47,789", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeDetailId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:47,790", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:38:49,699", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:49,699", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ServicePointCount"}
{"asctime": "2020-03-27 16:38:51,479", "levelname": "DEBUG", "message": "ApplyDataModel :Field ServicePointCount added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/ServingArea"}
{"asctime": "2020-03-27 16:38:51,479", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Structure', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'structure.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:38:51,707", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class Structure exists."}
{"asctime": "2020-03-27 16:38:51,708", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:38:53,093", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Structure"}
{"asctime": "2020-03-27 16:38:55,136", "levelname": "DEBUG", "message": "ApplyDataModel :Structure created"}
{"asctime": "2020-03-27 16:38:55,137", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:38:55,137", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:38:56,931", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:38:56,931", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureId"}
{"asctime": "2020-03-27 16:38:58,768", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:38:58,768", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeDetailId"}
{"asctime": "2020-03-27 16:39:00,496", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeDetailId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:00,496", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureType"}
{"asctime": "2020-03-27 16:39:02,207", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:02,207", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:39:03,994", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:03,994", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromCableId"}
{"asctime": "2020-03-27 16:39:05,710", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromCableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:05,711", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToCableId"}
{"asctime": "2020-03-27 16:39:07,461", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToCableId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:07,462", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerStructureId"}
{"asctime": "2020-03-27 16:39:09,169", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerStructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:09,169", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureAudit"}
{"asctime": "2020-03-27 16:39:11,107", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureAudit added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/Structure"}
{"asctime": "2020-03-27 16:39:11,107", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'SupportingStructure', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'supportingstructure.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:39:11,312", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class SupportingStructure exists."}
{"asctime": "2020-03-27 16:39:11,312", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:39:12,592", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class SupportingStructure"}
{"asctime": "2020-03-27 16:39:14,636", "levelname": "DEBUG", "message": "ApplyDataModel :SupportingStructure created"}
{"asctime": "2020-03-27 16:39:14,637", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:14,637", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:39:16,418", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:16,418", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-27 16:39:18,022", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:18,022", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SupportingStructureId"}
{"asctime": "2020-03-27 16:39:19,846", "levelname": "DEBUG", "message": "ApplyDataModel :Field SupportingStructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:19,847", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InstallationType"}
{"asctime": "2020-03-27 16:39:21,699", "levelname": "DEBUG", "message": "ApplyDataModel :Field InstallationType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:21,700", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromDistance"}
{"asctime": "2020-03-27 16:39:23,586", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromDistance added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:23,587", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToDistance"}
{"asctime": "2020-03-27 16:39:25,414", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToDistance added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:25,414", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Size"}
{"asctime": "2020-03-27 16:39:27,187", "levelname": "DEBUG", "message": "ApplyDataModel :Field Size added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/SupportingStructure"}
{"asctime": "2020-03-27 16:39:27,188", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'WorkArea', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'workarea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:39:27,423", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class WorkArea exists."}
{"asctime": "2020-03-27 16:39:27,423", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:39:28,807", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class WorkArea"}
{"asctime": "2020-03-27 16:39:30,944", "levelname": "DEBUG", "message": "ApplyDataModel :WorkArea created"}
{"asctime": "2020-03-27 16:39:30,945", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:30,946", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:39:32,638", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:32,638", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaCableType"}
{"asctime": "2020-03-27 16:39:34,317", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaCableType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:34,317", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaArchitecture"}
{"asctime": "2020-03-27 16:39:36,003", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaArchitecture added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:36,004", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-27 16:39:37,790", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:37,791", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-27 16:39:39,558", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:39,559", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-27 16:39:41,672", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:41,673", "levelname": "DEBUG", "message": "ApplyDataModel :Applying IsActive"}
{"asctime": "2020-03-27 16:39:43,813", "levelname": "DEBUG", "message": "ApplyDataModel :Field IsActive added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:43,814", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Status"}
{"asctime": "2020-03-27 16:39:46,036", "levelname": "DEBUG", "message": "ApplyDataModel :Field Status added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea"}
{"asctime": "2020-03-27 16:39:46,037", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'PowerEquipment', 'geometry': '', 'spatialref': '', 'fields': 'powerequipment.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:39:46,228", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class PowerEquipment exists."}
{"asctime": "2020-03-27 16:39:46,229", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:39:47,543", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table PowerEquipment"}
{"asctime": "2020-03-27 16:39:49,749", "levelname": "DEBUG", "message": "ApplyDataModel :PowerEquipment created"}
{"asctime": "2020-03-27 16:39:49,750", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PowerEquipment"}
{"asctime": "2020-03-27 16:39:49,751", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureId"}
{"asctime": "2020-03-27 16:39:51,147", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureId added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PowerEquipment"}
{"asctime": "2020-03-27 16:39:51,148", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentType"}
{"asctime": "2020-03-27 16:39:52,567", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentType added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PowerEquipment"}
{"asctime": "2020-03-27 16:39:52,568", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentCount"}
{"asctime": "2020-03-27 16:39:54,061", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentCount added to C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/PowerEquipment"}
{"asctime": "2020-03-27 16:39:54,061", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
{"asctime": "2020-03-27 16:40:39,117", "levelname": "DEBUG", "message": "ApplyDataModel :Setting subtype field C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea,domains failed"}
{"asctime": "2020-03-27 16:40:39,118", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000728: Field domains does not exist within table\nFailed to execute (SetSubtypeField).\n"}
{"asctime": "2020-03-27 16:40:39,118", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 136, in _process_subtypes\n    arcpy.SetSubtypeField_management(full_fc, fd)\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19657, in SetSubtypeField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19654, in SetSubtypeField\n    retval = convertArcObjectToPythonObject(gp.SetSubtypeField_management(*gp_fixargs((in_table, field, clear_value), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000728: Field domains does not exist within table\nFailed to execute (SetSubtypeField).\n"}
{"asctime": "2020-03-27 16:40:39,162", "levelname": "DEBUG", "message": "ApplyDataModel :Setting subtype code C:/Cloud/Dropbox/Work/Axin/rdtdesign.gdb/WorkArea,domains,1,{'AreaArchitecture': 'WorkAreaArchitecture', 'AreaCableType': 'WorkAreaCableType'} failed"}
{"asctime": "2020-03-27 16:40:39,162", "levelname": "ERROR", "message": "ApplyDataModel :Object: Error in executing tool"}
{"asctime": "2020-03-27 16:40:39,163", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 142, in _process_subtypes\n    arcpy.AddSubtype_management(full_fc, code, st_vals[code])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19589, in AddSubtype\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19586, in AddSubtype\n    retval = convertArcObjectToPythonObject(gp.AddSubtype_management(*gp_fixargs((in_table, subtype_code, subtype_description), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\nRuntimeError: Object: Error in executing tool"}
{"asctime": "2020-03-27 16:41:15,915", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain ApprovalStatus"}
{"asctime": "2020-03-27 16:41:15,915", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:41:15,915", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:41:20,386", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain RouteBuildType"}
{"asctime": "2020-03-27 16:41:20,386", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:41:20,387", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-27 16:41:21,854", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineChangeArea', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'redlinechangearea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:41:22,041", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineChangeArea exists."}
{"asctime": "2020-03-27 16:41:22,041", "levelname": "DEBUG", "message": "ApplyDataModel :We are NOT deleting - cannot create this feature class as it already exists."}
{"asctime": "2020-03-27 16:41:22,041", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlinePointMarker', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'redlinepointmarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-27 16:41:22,244", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlinePointMarker exists."}
{"asctime": "2020-03-27 16:41:22,244", "levelname": "DEBUG", "message": "ApplyDataModel :We are NOT deleting - cannot create this feature class as it already exists."}
{"asctime": "2020-03-27 16:41:22,244", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineLineMarker', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'redlinelinemarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:41:22,402", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineLineMarker exists."}
{"asctime": "2020-03-27 16:41:22,403", "levelname": "DEBUG", "message": "ApplyDataModel :We are NOT deleting - cannot create this feature class as it already exists."}
{"asctime": "2020-03-27 16:41:22,403", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineComment', 'geometry': '', 'spatialref': '', 'fields': 'redlinecomment.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-27 16:41:22,591", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineComment exists."}
{"asctime": "2020-03-27 16:41:22,592", "levelname": "DEBUG", "message": "ApplyDataModel :We are NOT deleting - cannot create this feature class as it already exists."}
{"asctime": "2020-03-27 16:41:22,592", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
{"asctime": "2020-03-27 16:41:23,990", "levelname": "DEBUG", "message": "ApplyDataModel :Can't add relationship for to ChangeArea_Comment"}
{"asctime": "2020-03-27 16:41:23,991", "levelname": "ERROR", "message": "ApplyDataModel :ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:23,991", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 88, in _process_relationship_classes\n    arcpy.CreateRelationshipClass_management(f\"{self._connection}/{a_relationship['origin']}\",f\"{self._connection}/{a_relationship['destination']}\", a_relationship['outclass'], a_relationship['type'], a_relationship['forwardlabel'], a_relationship['backlabel'], a_relationship['message'], a_relationship['cardinality'], a_relationship['attributed'], a_relationship['originpk'], a_relationship['originfk'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19096, in CreateRelationshipClass\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19093, in CreateRelationshipClass\n    retval = convertArcObjectToPythonObject(gp.CreateRelationshipClass_management(*gp_fixargs((origin_table, destination_table, out_relationship_class, relationship_type, forward_label, backward_label, message_direction, cardinality, attributed, origin_primary_key, origin_foreign_key, destination_primary_key, destination_foreign_key), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:25,229", "levelname": "DEBUG", "message": "ApplyDataModel :Can't add relationship for to PointMarker_Comment"}
{"asctime": "2020-03-27 16:41:25,230", "levelname": "ERROR", "message": "ApplyDataModel :ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:25,230", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 88, in _process_relationship_classes\n    arcpy.CreateRelationshipClass_management(f\"{self._connection}/{a_relationship['origin']}\",f\"{self._connection}/{a_relationship['destination']}\", a_relationship['outclass'], a_relationship['type'], a_relationship['forwardlabel'], a_relationship['backlabel'], a_relationship['message'], a_relationship['cardinality'], a_relationship['attributed'], a_relationship['originpk'], a_relationship['originfk'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19096, in CreateRelationshipClass\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19093, in CreateRelationshipClass\n    retval = convertArcObjectToPythonObject(gp.CreateRelationshipClass_management(*gp_fixargs((origin_table, destination_table, out_relationship_class, relationship_type, forward_label, backward_label, message_direction, cardinality, attributed, origin_primary_key, origin_foreign_key, destination_primary_key, destination_foreign_key), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:26,504", "levelname": "DEBUG", "message": "ApplyDataModel :Can't add relationship for to LineMarker_Comment"}
{"asctime": "2020-03-27 16:41:26,504", "levelname": "ERROR", "message": "ApplyDataModel :ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:26,505", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\ApplyDataModel\\ApplyDataModel.py\", line 88, in _process_relationship_classes\n    arcpy.CreateRelationshipClass_management(f\"{self._connection}/{a_relationship['origin']}\",f\"{self._connection}/{a_relationship['destination']}\", a_relationship['outclass'], a_relationship['type'], a_relationship['forwardlabel'], a_relationship['backlabel'], a_relationship['message'], a_relationship['cardinality'], a_relationship['attributed'], a_relationship['originpk'], a_relationship['originfk'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19096, in CreateRelationshipClass\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19093, in CreateRelationshipClass\n    retval = convertArcObjectToPythonObject(gp.CreateRelationshipClass_management(*gp_fixargs((origin_table, destination_table, out_relationship_class, relationship_type, forward_label, backward_label, message_direction, cardinality, attributed, origin_primary_key, origin_foreign_key, destination_primary_key, destination_foreign_key), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: ERROR 999999: Something unexpected caused the tool to fail. Contact Esri Technical Support (http://esriurl.com/support) to Report a Bug, and refer to the error help for potential solutions or workarounds.\nFailed to execute (CreateRelationshipClass).\n"}
{"asctime": "2020-03-27 16:41:26,506", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingAreaMarker', 'geometry': 'POLYGON', 'spatialref': 2277, 'fields': 'fieldingareamarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-27 16:41:26,693", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class FieldingAreaMarker exists."}
{"asctime": "2020-03-27 16:41:26,693", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:41:27,729", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingAreaMarker"}
{"asctime": "2020-03-27 16:41:29,524", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingAreaMarker created"}
{"asctime": "2020-03-27 16:41:29,525", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-27 16:41:29,525", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-27 16:41:31,127", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-27 16:41:31,127", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:41:32,600", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-27 16:41:32,601", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingPointMarker', 'geometry': 'POINT', 'spatialref': 2277, 'fields': 'fieldingpointmarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-27 16:41:32,786", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class FieldingPointMarker exists."}
{"asctime": "2020-03-27 16:41:32,787", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:41:33,848", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingPointMarker"}
{"asctime": "2020-03-27 16:41:35,563", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingPointMarker created"}
{"asctime": "2020-03-27 16:41:35,564", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingPointMarker"}
{"asctime": "2020-03-27 16:41:35,565", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-27 16:41:37,013", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingPointMarker"}
{"asctime": "2020-03-27 16:41:37,014", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:41:38,557", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingPointMarker"}
{"asctime": "2020-03-27 16:41:38,557", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingLineMarker', 'geometry': 'POLYLINE', 'spatialref': 2277, 'fields': 'fieldinglinemarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-27 16:41:38,745", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class FieldingLineMarker exists."}
{"asctime": "2020-03-27 16:41:38,746", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-27 16:41:39,836", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:41,664", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingLineMarker created"}
{"asctime": "2020-03-27 16:41:41,665", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:41,666", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-27 16:41:43,133", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:43,134", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-27 16:41:44,773", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:44,773", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-27 16:41:46,315", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:46,315", "levelname": "DEBUG", "message": "ApplyDataModel :Applying LineType"}
{"asctime": "2020-03-27 16:41:47,782", "levelname": "DEBUG", "message": "ApplyDataModel :Field LineType added to C:/Cloud/Dropbox/Work/Axin/rdtfielding.gdb/FieldingLineMarker"}
{"asctime": "2020-03-27 16:41:47,783", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
